CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## Unreleased
 
-## 0.9.3 - 2024-08-20
-
+## 0.9.4 - 2024-08-27
 ### Added
 * [[894]](https://github.com/microsoft/vscode-azureresourcegroups/pull/894) Add staged support for MongoClusters
 
+## 0.9.3 - 2024-08-20
+
+Skipped
```

Comment:
Should we write a little note as to why it was skipped? It probably doesn't really matter, but might be nice to document for future us.